selfdrive/locationd/locationd.cc

```diff
@@ -497,27 +498,29 @@ int Localizer::locationd_thread() {
   SubMaster sm(service_list, nullptr, { "gpsLocationExternal" });
 
   uint64_t cnt = 0;
+  bool filterValid = false;
```

Comment:
maybe call this filterInitialized instead? took me a second to figure out that's what it was